traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.7.2
+version: 8.8.1
```

Comment:
it should be 8.9.0, no? It's adding new functionality so for me, its a minor and not a patch

Reply:
Minor was already increased in this PR from 8.7.0 to 8.8.0 due to this new feature. I increase the patch just to fix the ci :smile:

Reply:
Updated due to master